gui/gui.md

```diff
@@ -91,11 +87,25 @@ The third tab, "Preferences", allows you to change some default behaviors of Gra
 
 The "Home Page" dropdown menu selects which page will be displayed after logging in or clicking the Gravwell icon next to the main menu. By default, the new search page is shown, but you can chose to be shown a list of dashboards, kits, or playbooks instead.
 
+The "Left menu default visibility" dropdown sets the default visibility of the navigation menu. 
+
+The "Interface theme" dropdown is of particular interest, as it selects a GUI-wide color scheme (including the ever-popular dark modes). 
+
+The "24-Hour format" toggle enables users to change the time format to 24-hour instead of the the 12-hour AM/PM format.
+
+The "Chart theme" dropdown selects different color palettes which will be used when drawing charts. 
+
+The "Point-to-point map theme" dropdown selects the color theme to use with point-to-point maps.
+
+The "Editor theme" & "Font size" options control the appearance of Gravwell's built-in text editor, which is used to create automation scripts and in a few other places.
+
 The "Search Group Visibility" option allows you to share the results of all searches with a given group; this can be a convenient way to collaborate. In the screenshot, the user has selected the group named "foo"; all members of that group will have access to the searches this user runs in the future.
 
 The "Advanced Preferences" section can be ignored by most users. Selecting "Developer mode" enables manual editing of JSON preferences (see [this page](/configuration/gui) for more information), while toggling "Experimental Features" will enable the Experimental Features section in the main menu.
 
 ### Email Server
 
-The final tab, "Email Server", is extremely important for users who intend to use Gravwell to send automated email alerts.  Complete documentation is available on the [Email Configuration](/configuration/email) page.
+The fourth tab, "Email Server", is extremely important for users who intend to use Gravwell to send automated email alerts.  Complete documentation is available on the [Email Configuration](/configuration/email) page.
 
+### Capabilities
+The last tab shows a list of capabilities when Capability Based Access Control is enabled. Complete documentation is available on the [CBAC](/cbac/cbac) page.
```

Comment:
Maybe this should say list of ***granted*** capabilities?

Reply:
That's not a bad idea. I think it clarifies what the table expresses.